go/teams/list.go

```diff
@@ -52,21 +52,32 @@ func List(ctx context.Context, g *libkb.GlobalContext, arg keybase1.TeamListArg)
 		uid = res.GetUID()
 	}
 
+	me, err := libkb.LoadMe(libkb.NewLoadUserArg(g))
```

Comment:
You just need this for the current user's UID, correct?  If so,

    g.ActiveDevice.UID()

will be less resource-intensive...